test/integration/nginx/Dockerfile

```diff
@@ -59,7 +73,7 @@ RUN mkdir -p /var/www/
 COPY ./test/integration/nginx/index.html /var/www/index.html
 
 # Get Wiremock
-ADD http://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.18.0/wiremock-standalone-2.18.0.jar wiremock-standalone-2.18.0.jar
+RUN wget http://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.18.0/wiremock-standalone-2.18.0.jar -O wiremock-standalone-2.18.0.jar
```

Comment:
Any special reason for this?

Reply:
Docker will cache `RUN wget` but will execute every single ADD (and check the checksum to see whether to bust the cache). This makes it slow, and makes me feel bad for the webmaster of whoever serves me files.

So for files that I expect to be immutable, RUN wget is nicer.